derp/xdp/xdp_linux.go

```diff
@@ -262,6 +266,31 @@ func (s *STUNServer) Collect(metricCh chan<- prometheus.Metric) {
 	s.metrics.registry.Collect(metricCh)
 }
 
+func (s *STUNServer) SetDropSTUN(v bool) error {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+	dropSTUN := 0
+	if v {
+		dropSTUN = 1
+	}
+	xdpConfig := bpfConfig{
+		DstPort:  uint16(s.dstPort),
+		DropStun: uint16(dropSTUN),
+	}
+	var key uint32
+	err := s.objs.ConfigMap.Put(key, &xdpConfig)
```

Comment:
I believe there's a slight preference to make `xdpConfig` a pointer at initialization if it's only ever used as a pointer.

Reply:
updated in https://github.com/tailscale/tailscale/pull/12527/commits/41b5d9932dfca4bb99c8c04add93cc9ae19d34be